python_modules/dagster/dagster/_utils/typing_api.py

```diff
@@ -128,3 +128,17 @@ def is_typing_type(ttype):
         or ttype is typing.Dict
         or ttype is typing.List
     )
+
+
+def flatten_unions(annotation: typing.Type) -> typing.Iterable[typing.Type]:
```

Comment:
Great. This name is more obvious I think.

Reply:
minor nit: should this return a set?

Reply:
> minor nit: should this return a set?

👍  - changed